### DISCUSSION TITLE
Chat into AxelChat to all "source"

### DISCUSSION THREAD

**Discussion by hadesfury** [3dproger/AxelChat#521] [Ideas]
How hard is it to allow to chat into AxelChat and forward the messages into all connected chats?

**Comment by 3dproger** [comment]
Hi,

Unfortunately, it is difficult to do this for all chats, since not all services provide the ability to send messages without any obstacles. The exception is Twitch, Trovo and, perhaps, a couple more services. Therefore, I can do this only for some services, at least for now

**Reply by hadesfury** [reply]
Sure, it makes it less interesting as a "feature"

Thank you for the answer

**Reply by Aeon1704** [reply]
Will be very useful for my attempt to integrate VKVideo Live into Streamer Bot using AxelChat, if AxelChat will have ability to send message in VKVideo Live :)
